### ISSUE TITLE
KCD 2020 Nordics, Stockholm, Sweden

### ISSUE THREAD

**Issue by kaspernissen** [cncf/kubernetes-community-days#232]
### Submission checklist:

*Please check each of these as you complete them:

* [x] No other Kubernetes Community Days event in the same city
* [x] Minimum of 3 organizers from 3 different organizations
* [x] Each organizer lives in or just outside of the KCD location
* [x] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer
* [x] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [ ] Minimum of 3 sponsors committed
* [ ] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
* [x] Agree to use Eventbrite for registration
* [x] Agree to use Sched for organizing the program page
* [ ] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel

### Organizer 1: 
Manuela Fredriksson, ELASTX, Title, Email, CNCF Slack Username
### Organizer 2: 
Henrik Enberg, Elastisys, Operations Manager, Email, CNCF Slack Username
### Organizer 3: 
Stefano Cucchiella, City Network, Solutions Architect, Email, CNCF Slack Username
### Organizer 4: 
Kasper Nissen, LUNAR, Site Reliability Engineer, kaspernissen@gmail.com, phennex (CNCF Ambassador)
### Organizer 5: 
Rasmus Steiniche, Neuro space, CEO, Email, CNCF Slack Username
### Organizer 6: 
Jessica Andersson, Meltwater, Infrastructure Engineer, Email, CNCF Slack Username

### Sponsor 1
TBD
### Sponsor 2
TBD
### Sponsor 3
TBD

### Fiscal sponsor that will accept payments
TBD

**Comment by KTan1226** [comment]
Great start @kaspernissen! Please update the issue as you complete the items on the checklist. Once everything is complete, we'll sign off on your team creating the event site.

**Comment by kaspernissen** [comment]
@KTan1226 we have now completed the submission checklist and will be ready to move forward 🎉 

We are looking at the dates 27th and 28th of October 2020.

**Comment by KTan1226** [comment]
Great @kaspernissen! You are approved to build your site. I've added all organizers to the KCD-organizers Slack channel and created the alias (organizers-stockholm@kubernetescommunitydays.org) for your use. I will send a test of the alias shortly.

Can you also confirm the Twitter account for your event? Thank you!

**Comment by kaspernissen** [comment]
Thank you very much, @KTan1226! 

We will be using this twitter profile (I just created it and will be filling in more content tomorrow): https://twitter.com/kcdnordics

Invitations received 👍

**Comment by kaspernissen** [comment]
We have decided to postpone the Cloud Native Nordics Kubernetes Community Day from October 2020 to Febraury 2021. 

The reasons are as follows;
* Corona virus panic means a lot of events get cancelled and we are concerned about how it will impact our ability to sign sponsors and sell tickets
* with that concern we have to assess the monetary risk of moving forward with the event right now, because if we do we committ to paying 50% to the venue if we don’t cancel within a month. Right now it seems unclear if we know more in a month.
* in addition this adds an extra stress on the organisers especially since we have 0 money in our non-profit currently and would be very sensitive to cancel late in the process. 

With this as a background we decided to postpone the conference and cancel planning meetings until April 14th where we will pick up again an reassess the situation.

**Comment by KTan1226** [comment]
Thanks for the update @kaspernissen. Please keep us updated.
